client-go/cmd/utils.go

```diff
@@ -66,3 +67,13 @@ func load(appID string) (*client.Client, string, error) {
 
 	return c, appID, nil
 }
+
+func drinkOfChoice() string {
```

Comment:
And to think our BDFL once wanted to get rid of this "unprofessional" message he himself wrote! Thanks to @bacongobbler and you, it lives on. See #2393 for some laughs. :coffee: